contrib/download-frozen-image-v2.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
```

Comment:
Oh, good point!  I think we're somewhat inconsistent on this point:
```console
$ git grep -n '^#!/bin/bash' | wc -l
88
$ git grep -n '^#!/usr/bin/env bash' | wc -l
18
```

Maybe swapping all those would be a good follow-up PR?